src/components/custom/_signup-new.tsx

```diff
@@ -116,42 +116,24 @@ const SocialButton = styled(Button)<SocialButtonProps>`
     display: inline-flex;
     justify-content: center;
     align-items: center;
+    border: none;
     box-shadow: none;
     background-color: ${(props) => props.bgColor || 'var(--color-white)'};
-    border: solid 1px var(--color-grey-21);
-    width: 12.5rem;
     height: 3.8rem;
-    padding: 0.5rem 0;
-
-    &:hover {
-        background: ${(props) => {
-            if (props.provider === 'facebook') return 'var(--color-grey-4)'
-        }};
-    }
-
-    @media ${device.tabletL} {
-        width: 100%;
-        height: 6rem;
-        margin-top: 1rem;
-    }
-    @media ${device.mobileM} {
-        &:first-child {
-            margin-right: 1.2rem;
-        }
-    }
+    padding: 0;
 `
 const SocialWrapper = styled.div<CSSProperties>`
     width: 100%;
     margin-top: 2.4rem;
-    display: flex;
-    justify-content: space-between;
+    display: grid;
+    grid-template-columns: repeat(3, max-content);
+    gap: ${(props) => props.gap || 0};
```

Comment:
nitpick
```suggestion
    gap: ${({gap}) => gap};
```

Reply:
I will just put it as `props` instead of destruct the object, following the approach in other css 😄 . But, will remove `|| 0`